src/test/java/wooteco/subway/station/StationAcceptanceTest.java

```diff
@@ -1,138 +1,140 @@
+            .collect(Collectors.toList());
+
+        // when
+        List<Long> retrievedStationIds = requestAndGetAllSavedStationIds();
 
+        // then
+        assertThat(retrievedStationIds).containsExactlyInAnyOrderElementsOf(createdStationIds);
+    }
+
+    private List<Long> requestAndGetAllSavedStationIds() {
+        return requestAndGetAllSavedStationResponseDtos().stream()
+            .map(StationResponseDto::getId)
+            .collect(Collectors.toList());
+    }
+
+    private List<StationResponseDto> requestAndGetAllSavedStationResponseDtos() {
         // when
         ExtractableResponse<Response> response = RestAssured.given().log().all()
-                .when()
-                .get("/stations")
-                .then().log().all()
-                .extract();
+            .when()
+            .get("/stations")
+            .then().log().all()
+            .extract();
 
         // then
         assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
-        List<Long> expectedLineIds = Arrays.asList(createResponse1, createResponse2).stream()
-                .map(it -> Long.parseLong(it.header("Location").split("/")[2]))
-                .collect(Collectors.toList());
-        List<Long> resultLineIds = response.jsonPath().getList(".", StationResponse.class).stream()
-                .map(it -> it.getId())
-                .collect(Collectors.toList());
-        assertThat(resultLineIds).containsAll(expectedLineIds);
+        assertThat(response.contentType()).isEqualTo(MediaType.APPLICATION_JSON_VALUE);
+
+        return response.jsonPath().getList(".", StationResponseDto.class);
     }
 
-    @DisplayName("지하철역을 제거한다.")
+    @DisplayName("지하철역을 Id로 제거한다.")
     @Test
-    void deleteStation() {
+    void deleteStationById() {
         // given
-        Map<String, String> params = new HashMap<>();
-        params.put("name", "강남역");
-        ExtractableResponse<Response> createResponse = RestAssured.given().log().all()
-                .body(params)
-                .contentType(MediaType.APPLICATION_JSON_VALUE)
-                .when()
-                .post("/stations")
-                .then().log().all()
-                .extract();
+        requestCreateStationWithNameAndGetResponse("강남역");
+        requestCreateStationWithNameAndGetResponse("역삼역");
+        List<Long> allSavedStationIdsBeforeDelete = requestAndGetAllSavedStationIds();
+        Long stationIdToDelete = allSavedStationIdsBeforeDelete.get(0);
+        Long stationIdNotToDelete = allSavedStationIdsBeforeDelete.get(1);
 
         // when
-        String uri = createResponse.header("Location");
         ExtractableResponse<Response> response = RestAssured.given().log().all()
-                .when()
-                .delete(uri)
-                .then().log().all()
-                .extract();
+            .when()
+            .delete("/stations/" + stationIdToDelete)
```

Comment:
```suggestion
            .delete("/stations/{stationId}", stationIdToDelete)
```

정확한 테스트 결과 확인을 위해 위와 같이 작성해보면 어떨까요!?

Reply:
오, 이런 방식도 가능하군요. 더욱 명시적이고 깔끔하네요!! 꿀팁 감사합니다!! 😆